src/main/scala/com.mediative.eigenflow/EigenflowBootstrap.scala

```diff
@@ -47,17 +44,5 @@ trait EigenflowBootstrap {
   implicit val system = ActorSystem("DataFlow", ConfigurationLoader.config)
 
   // create main actor and tell to proceed.
-  system.actorOf(Props(new ProcessManager(process, startDate, stopSystem _))) ! Continue
-
-  /**
-   * System shutdown logic.
-   */
-  private def stopSystem(code: Int): Unit = {
```

Comment:
See the comment I have below.  I would keep this stopSystem here, and instead pass the function into the supervisor.  I prefer that pattern because it means the thing that creates the messaging system and actor system will also have the responsibility of shutting it down.